lab_2_retrieval_w_bm25/main.py

```diff
@@ -33,6 +53,16 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+    right_input = isinstance(tokens, list) and isinstance(stopwords, list) and all(
+        isinstance(token, str) for token in tokens) and all(
+        isinstance(word, str) for word in stopwords) and tokens != [] and stopwords != []
```

Comment:
`tokens != []` is a bad check. Use falsy/truthy aproach https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-how-is-it-different-from-true-and-false

Reply:
done